CONTRIBUTING.md

```diff
@@ -126,6 +124,22 @@ your clone of the originating `gh-pages` branch.
 Lastly, published copies of all the lessons are available in the `gh-pages` branch of the originating
 repository for reference while revising.
 
+## Previewing your changes locally
+
+Extensive instructions for building and viewing the pages locally can be found [here](https://carpentries.github.io/lesson-example/setup.html). To get started quickly, run
+```bash
+sudo apt install ruby-dev ruby-bundler
+```
```

Comment:
Maybe we need to widen this to include other sources? I'm on fedora, for instance. 

```suggestion
```bash
sudo apt install ruby-dev ruby-bundler
```
or 
```bash
 sudo dnf install rubygem-bundler ruby-devel
```
```

Reply:
I guess adding instructions for more operating systems is great. Could you add what's needed for fedora?

Reply:
Perhaps instead we could make a `environment.yml` conda for the tutorial?

Reply:
I'm not sure how much ruby packages there are on conda. If you want something os independent, we could mention the `make docker-serve` command? That also works fine, but then you have to install docker first.